exercises/04-integration/after/Divergent.Customers.Data/Divergent.Customers.Data.csproj

```diff
@@ -1,72 +1,19 @@
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <TargetFramework>net461</TargetFramework>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="EntityFramework, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
-      <HintPath>..\packages\EntityFramework.6.1.3\lib\net45\EntityFramework.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="EntityFramework.SqlServer, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL">
-      <HintPath>..\packages\EntityFramework.6.1.3\lib\net45\EntityFramework.SqlServer.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-    <ProjectReference Include="..\Divergent.ITOps.Interfaces\Divergent.ITOps.Interfaces.csproj">
-      <Project>{23dea9ff-fcb9-4c9f-94f7-0a77e23d8791}</Project>
-      <Name>Divergent.ITOps.Interfaces</Name>
-    </ProjectReference>
+    <PackageReference Include="EntityFramework" Version="6.1.3" />
   </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="Context\CustomersContext.cs" />
-    <Compile Include="ITOps\CustomerInfoProvider.cs" />
-    <Compile Include="Migrations\DatabaseInitializer.cs" />
-    <Compile Include="Migrations\SeedData.cs" />
-    <Compile Include="Models\Customer.cs" />
-    <Compile Include="Models\Order.cs" />
-    <None Include="App.config" />
-    <None Include="packages.config" />
+    <ProjectReference Include="..\Divergent.ITOps.Interfaces\Divergent.ITOps.Interfaces.csproj" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+
   <PropertyGroup>
-    <PostBuildEvent>copy /Y "$(TargetDir)$(ProjectName).dll" "$(SolutionDir)Divergent.ITOps\Providers\$(ProjectName).dll"</PostBuildEvent>
+    <PostBuildEvent>copy /Y "$(TargetDir)Divergent.Customers.Data.dll" "$(SolutionDir)Divergent.ITOps\Providers\Divergent.Customers.Data.dll"</PostBuildEvent>
```

Comment:
Was this change required because of the bug in post build events with `$(ProjectName)`?

Reply:
@mauroservienti for one reason or another, `$(ProjectName)` is empty when using SDK csproj. I don't know if this is a bug or by design, but I thought the easiest way to work around it is to hardcode the project name. It's unlikely to change and if it ever does, we'll see a problem straight away when smoke testing the exercise.

BTW - I had to do the same thing in `Divergent.Shipping.Data.csproj`.

Reply:
It’s a bug, I faced it a few months ago, basically with the new project style not all variables are filled correctly before the execution of pre/post build steps.